apps/web/components/dashboard/EditableText.tsx

```diff
@@ -96,7 +96,7 @@ function ViewMode({
 }: Props) {
   return (
     <Tooltip delayDuration={500}>
-      <div className="flex items-center gap-3 text-center">
+      <div className="flex max-w-full items-center gap-3 text-center">
```

Comment:
Ah, good catch--I didn't even notice the tooltip there! I ended up using `break-words` to ensure that more typical titles with multiple words don't break in strange places. The editable title field still uses `break-all`, since where the lines break doesn't matter so much there, as long as the whole title is visible!